### DISCUSSION TITLE
Write to hive partitioned parquet files

### DISCUSSION THREAD

**Discussion by adriangb** [duckdb/duckdb#4476] [Ideas]
This would be really nice when reading from blob storage and writing back out to it. I imagine exporting partitions concurrently in multiple threads will be a huge win for network bottlenecks.

**Comment by cboettig** [comment]
This would be huge.  Since arrow supports this already, I wonder if there's a mechanism to piggyback on the arrow connection?  (Though we've also found native duckdb performance against blob storage to be significantly faster)

**Reply by ned2** [reply]
Writing Hive partitioned Parquet (and CSV) is already supported! https://duckdb.org/docs/data/partitioning/partitioned_writes#partitioned-writes

**Reply by cboettig** [reply]
@ned2 Thanks, this is great news!

Minor observation: https://duckdb.org/docs/extensions/httpfs seems to suggest that overwrites use the notation  `ALLOW_OVERWRITE TRUE`, whereas in my testing they appear to use the same notation as shown under partitioning documentation you link, just adding the flag ` OVERWRITE_OR_IGNORE`.  Maybe the docs are just out of date?  

Anyway working great for me so far!  many thanks!
